generator/generateV2_service.php

```diff
@@ -0,0 +1,92 @@
+<?php
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+require_once __DIR__ . '/Google/Service/Generator/Method.php';
```

Comment:
Does it make sense to add a psr-4 autoload entry in `composser.json`? That way we don't need the `require_once` calls any more.

Reply:
Definitely a change I want to put in v3.